feature/bgp/policybase/otg_tests/default_policies_test/bgp_default_policies_test.go

```diff
@@ -584,55 +586,20 @@ func deleteBGPPolicy(t *testing.T, dut *ondatra.DUTDevice, nbrList []*bgpNbrList
 }
 
 func configureRoutingPolicyDefaultAction(t *testing.T, dut *ondatra.DUTDevice, action string, replace bool) {
```

Comment:
This function was intended to configure the default-action under the routing-policy, substituting bgp default-policy with policy default-action using the deviation.

Below changes to this function seems to be configuring the regular routing-policy with action (not default-action), considering different behavior should it a different function for this ?

Reply:
The change is creating a single action with no match criteria to a default accept or reject when "bgp_default_policy_unsupported" deviation is set ?  isn't that the expected configuration for implementations that do not support default policy action ?

Reply:
This is impacting testcases `7.1.1` and `7.1.2` in this script, which requires both default-action/default-policy and ebgp/ibgp policy to exist together. Keeping these 2 cases in consideration, `configureRoutingPolicyDefaultAction` was written, which configures the `default-action` in routing-policy (when `default-policy` in bgp is unsupported) along with existing ebgp/ibgp policy.

Reply:
With this change, we should have both the default-action/default-policy and ebgp/ibgp policy to exist together for 7.1.1 and 7.1.2 test case right ?
For both these test cases, we only do a gNMI.Update since replaceFlag is set to false.

Am i missing something here ?

Reply:
I see based on the changes to this function, it is not configuring the default-action anywhere. It is only configuring the policy which is intended for ebgp/ibgp

https://github.com/openconfig/featureprofiles/blob/53a90c708a5c265b23dbf9859fb5c150e7ebf56f/feature/bgp/policybase/otg_tests/default_policies_test/bgp_default_policies_test.go#L588-L604

For example, based on test `7.1.1`, default-action/default-policy is supposed to have `reject` action, and the ebpg/ibgp is supposed to have `accept` action, this cannot be achieved with above function - 

https://github.com/openconfig/featureprofiles/blob/53a90c708a5c265b23dbf9859fb5c150e7ebf56f/feature/bgp/policybase/otg_tests/default_policies_test/README.md?plain=1#L26-L28

Reply:
Hi Tushar,

"I see based on the changes to this function, it is not configuring the default-action anywhere"
What do you mean by default-action here ? Are you referring to an additional statement in the policy-definition that will have REJECT action for 7.1.1 and ACCEPT for 7.1.2. If so, that is exactly what this function is doing ( setting the OC path 'routing-policy/defined-sets/policy-definitions/policy-definition[name]/statements/statement[name]/actions/policy-result' ).  For implementations that do not support default import/export bgp policy, this is the expected deviation ( adding a default statement in the policy definition that will have an accept or reject definition ).

Please clarify what you mean by default-action not being set as part of this function.

Reply:
`default-action` in routing-policy is something similar to `default-policy` in BGP that we support.  However, as highlighted in the https://github.com/openconfig/featureprofiles/pull/3024#issuecomment-2150765971 , routing-policy is more like a helper-function and is not the focus of this test. So we using deviation for BGP config to avoid unsupported default-policy configuration with this approach also seems fine.

I am good with this change.